src/client/renderer/r_bsp.c

```diff
@@ -320,7 +320,7 @@ void R_DrawBspLeafs(void) {
 			if ((*s)->vis_frame != r_locals.vis_frame)
 				continue;
 
-			glDrawArrays(GL_POLYGON, (*s)->index, (*s)->num_edges);
+			R_DrawArrays(GL_TRIANGLE_FAN, (*s)->index, (*s)->num_edges);
 		}
```

Comment:
This is great. After this refactor, I'm eager to tear into the renderer and move world geometry (at least) to indexed arrays, so we'll have R_DrawElements as well. I dig it.